getspice.py

```diff
@@ -1,9 +1,7 @@
-#!/usr/bin/env python3
-
 """
 The MIT License (MIT)
 
-Copyright (c) [2015-2017] [Andrew Annex]
+Copyright (c) [2017] [ODC Space]
```

Comment:
no, this goes against the terms of the  MIT license. You need to append the ODC Space license to the end of the lists here (aka after the Philipp Rasch part), leaving 3 licenses in the header.